IOMC/RandomEngine/test/testRandomServiceTest4_cfg.py

```diff
@@ -36,6 +36,10 @@
         initialSeed = cms.untracked.uint32(7)
     ),
     t5 = cms.PSet(
+        engineName = cms.untracked.string('XorShift128Plus'),
+        initialSeedSet = cms.untracked.vuint32(7, 8, 9, 10)
```

Comment:
One question about this engine. Lets say you make 2 engines. Is it
OK for one engine to have seeds (7, 8, 9, 10) and the other to have
seeds (8, 8, 9, 10)? Will they give statistically independent sequences?
Are there any special rules about selecting seeds?

I am am looking at these tests again, but it has been a few years since
I went through these last and the tests are more complicated than the
service ...